tests/std/tests/P1425R6_queue_stack_constructors/test.cpp

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <deque>
+#include <forward_list>
+#include <iostream>
+#include <memory>
+#include <queue>
+#include <set>
+#include <stack>
+#include <type_traits>
+#include <utility>
+#include <vector>
+using namespace std;
+
+constexpr int some_data[]       = {0, 1, 2, 3, 4, 5};
+constexpr int additional_data[] = {6, 7, 8, 9, 10, 11};
+
+template <typename T>
+struct custom_allocator {
+    using value_type = T;
+
+    custom_allocator() noexcept = default;
+    template <typename U>
+    custom_allocator(const custom_allocator<U>&) noexcept {}
+
+    T* allocate(const size_t n) {
+        return allocator<T>{}.allocate(n);
+    }
+
+    void deallocate(T* const p, const size_t n) noexcept {
+        allocator<T>{}.deallocate(p, n);
+    }
+
+    template <class... Args>
+    void construct(T* const p, Args&&... args) {

```

Comment:
I believe that `construct` can be omitted - the library will provide default machinery, and nothing in this test appears to need this function.